src/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './App.css';
+
+class Header extends React.Component {
+  render() {
+    return (
+      <>
+        <h1>
+          Horned Beasts
```

Comment:
Another way to do this is to pass your title down as a prop. As far as use cases, this works just as well, but just as a way to practice props :)